samples/ComponentExamples/src/examples/ComponentExample.tsx

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { MessageThread, SendBox, usePropsFor } from '@azure/communication-react';
+import * as React from 'react';
+
+const messageContainerStyle = {
+  height: '8rem',
+  OverflowY: 'scroll',
+  border: 'solid 2px'
+};
+
+export const ComponentExample = (): JSX.Element => {
+  // usePropsFor will get all the pros and handlers to run the component
```

Comment:
typo: pros -> props

Also, handlers are also props. So better just say "will get all the props (including handlers) to"